src/components/helpers/form-helpers/parseEventDate.js

```diff
@@ -0,0 +1,7 @@
+import moment from "moment";
+
+const parseEventDate = value => {
+  return moment(value).format("ddd, MMM D, Y, LT");
+};
+
+export default parseEventDate;
```

Comment:
export the function as non-default, so file can contain other functions

Reply:
if i have only one function eslint say "use default"